### DISCUSSION TITLE
Using Cebra embeddings for further unsupervised clustering

### DISCUSSION THREAD

**Discussion by JohannaMankel** [AdaptiveMotorControlLab/CEBRA#196] [Q&A] [answered]
Hi, I was wondering if the embeddings generated by a cebra-model purely trained on behavioral features derived from DLC keypoints (like velocity/angle) can be used for further unsupervised clustering.
Currently, I'm using UMAP for dimensionality reduction followed by hdbscan but I'm trying to improve the clustering by using an approach that takes the temporal structure of the data into account.

Thanks for developing the tool and all the help,
Johanna

**Comment by MMathisLab** [comment, marked as answer]
Yes; in fact we did this already in another paper: Ye et al 2024 Nature Communications. https://www.nature.com/articles/s41467-024-48792-2/figures/4
